models/domains_tlds.php

```diff
@@ -1403,6 +1403,73 @@ public function getDomainsCompanySettings($company_id = null)
         return $domains_settings;
     }
 
+    /**
+     * Imports the TLDs and their pricing from a registrar module, if available
+     *
+     * @param array $tlds A list containing the TLDs to import from the registrar module
+     * @param int $module_id The ID of the registrar module
+     * @param int $company_id The ID of the company to import the TLDs
+     * @return bool True if all the TLDs where imported successfully, false otherwise
+     */
+    public function import(array $tlds, int $module_id, int $company_id = null) : bool
+    {
+        Loader::loadModels($this, ['ModuleManager']);
+
+        $company_id = !is_null($company_id) ? $company_id : Configure::get('Blesta.company_id');
+
+        // Format TLDS
+        $tlds = array_keys($tlds);
+
+        // Initialize module
+        $module = $this->ModuleManager->initModule($module_id);
+
+        if (!$module) {
+            return false;
+        }
+
+        // Import TLDs
+        if (!empty($tlds)) {
+            $module->setModuleRow($module->getModuleRows()[0] ?? null);
+
+            // Get TLD pricing, if available
+            $tld_pricing = $module->getTldPricing();
```

Comment:
I don't think this is annoying

Reply:
I think I don't understand, What do you mean by annoying? 🤔

Reply:
Not sure why I wrote that.  I meant "I don't think this is necessary"